Meta/prepare-code-coverage-artifact.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+'''
+Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+See https://llvm.org/LICENSE.txt for license information.
+
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+NOTE: This helper is copied from llvm-project/llvm/utils
+'''
```

Comment:
Can we `wget` this file or something instead of adding 3rd party stuff to the repo?

Reply:
yep sure can! finally got around to this. now doing a wget from the llvm project's github instead.